data/core/units/elves/Sharpshooter.cfg

```diff
@@ -17,7 +17,7 @@
     {AMLA_DEFAULT}
     cost=51
     usage=archer
-    description= _ "There was a legend of an elf who stopped an arrow by shooting it out of the air with one of his own. It is a testament to the skill of the elves that such a story could be taken seriously. The sharpshooters of the elves have, quite simply, mastered the art of archery."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
+    description= _ "Though not a formal title, ‘Sharpshooter’ is the epithet given to the most gifted Elvish marksmen. These expert archers are capable of feats that, even by Elven standards, border on miraculous: they are precise enough to hit enemy projectiles midair with their own arrows and can even shoot multiple projectiles simultaneously while maintaining enough power to damage steel armor. A single volley of their arrows is enough to bring down a knight on horseback, and even heavily-armored infantry will not survive more than a few additional shots. The sharpshooters of the elves have honed their art to its highest form and are the undisputed masters of archery."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MARKSMAN}
```

Comment:
I'm not happy with the part about multiple projectiles. How about this instead?

> These expert archers are capable of feats that, even by Elven standards, border on miraculous: their aim is precise enough to hit enemy projectiles midair, their draw is strong enough to damage steel armor, and their reload is fast enough to hit a man thrice before he hits the ground.

Reply:
I'm not a big fan of the three-pronged parallel construction (basically, these units have traits A, B, and C things going for them), and I feel that the part about the reload is a little weak. However, if someone else also prefers josteph's description over the one I currently have committed, please do say so and I will commit this version instead.

(I've already talked to josteph about this, just putting this here for everyone else to see.)

Reply:
I don't see the problem with the three-pronged parallel construction but also don't see the problem with multiple projectiles, as that is kind of a fantasy staple if I'm not mistaken. I suppose the only worry might be whether it contradicts the game mechanics?

Reply:
Yes, exactly. There's no equivalent in game mechanics for "shoots multiple projectiles simultaneously".

Reply:
Do they get more attacks per round? If so, then "can shoot three times where less skilled archer might only fire once" might be better

Reply:
They get more shots per attack, yes (5x4, 9x4, 10x5 at lvl1,lvl2,lvl3 respectively). I think it works better with an example ("before he hits the ground") than with an abstract statement, though it doesn't have to be that specific example. In fact I think it's not even that good an example, it's _too_ fast.